docs/getting-started.md

```diff
@@ -133,29 +169,172 @@ account, for developing your own study:
    work-in-progress changes.)
 8. Click **Create repository from template**
 
-### Clone it to your computer
+## 3. Setup the required software
 
-Now you'll want to get a copy of that code on your computer, in order to start developing your own study.
-
-1. Look above the list of files in your new repo for a **Code** button. ![image](images/code-button.png)
-2. Click **Open with GitHub Desktop** to clone and open the repository with
-   GitHub Desktop. ![image](images/open-with-desktop.png)
-3. Click **Choose...** and, using Windows Explorer, navigate to a local path
-   where you want to clone the repository.
-   ![image](images/clone-choose-button-url-win.png)
-4. Click **Clone**  ![image](images/clone-button-url-win.png)
+<!-- The generated table of contents does not dynamically adjust according to selected tab for tabbed contents. Use actual heading tags to prevent headings in tabbed sections from appearing in the table of contents. -->
+=== "Web browser (online)"
+
+    <h3>Open your repository with Gitpod</h3>
+
+    For the repository you just created, it will have a URL of the form:
+    `https://github.com/<your_username>/<your_research_repository>`
+    where the username and repository name are as you chose.
+
+    Place `https://gitpod.io/#` in front of that URL:
+
+    `https://gitpod.io/#https://github.com/<your_username>/<your_research_repository>`
+
+    and visit that URL. If it is the first time you have used Gitpod,
+    you'll see a "Log in" screen. You can use your GitHub account to
+    login to Gitpod: click "Continue with GitHub".
+
+    A Gitpod workspace containing a code editor, defaulting to Visual
+    Studio Code and command-line interface "terminal" should then
+    appear. The terminal that Gitpod provides runs commands on a Linux
+    computer, but, don't worry, the few command-line instructions for
+    working with OpenSAFELY are the same regardless of using Windows,
+    Linux or macOS.
+
+    <h3>Install **opensafely**</h3>
+
+    The `opensafely` software should already be installed if you start
+    Gitpod for the OpenSAFELY research template.
+
+    You can confirm this by typing `opensafely` in the terminal and
+    pressing ++enter++. You should see an output that starts:
+
+    ```sh
+    $ opensafely
+    usage: opensafely [-h] [--version] COMMAND ...
+    …
+    ```
+
+    <h3>How Gitpod workspaces operate</h3>
+
```

Comment:
There aren't any serious consequences. I agree that it distracts from the actual task at hand.

I don't know how obvious Gitpod is to other people. I know that how workspaces operate wasn't immediately obvious, because you don't see the list of workspaces when you open one up. We could move this inside the Reference section, or potentially drop it if it's not useful.